core-web/apps/dotcms-ui/src/app/portlets/dot-containers/dot-container-create/dot-container-properties/dot-container-properties.component.ts

```diff
@@ -131,7 +131,7 @@ export class DotContainerPropertiesComponent implements OnInit, AfterViewInit {
                 ({ code, structureId, containerId, containerInode, contentTypeVar }) => {
                     (this.form.get('containerStructures') as FormArray).push(
                         this.fb.group({
-                            code: new FormControl(code, [Validators.required]),
```

Comment:
If the `save button` is disabled when this control is required, can we test that the `code` form control does not have `[Validators.required]`? Otherwise, we can leave it without `tests`.

Reply:
updated